src/auth.ts

```diff
@@ -167,6 +172,7 @@ export async function notifyAuthSuccess(): Promise<void> {
         return;
     }
     try {
+        getPreauthInfo();
         const sessionInfo = await getSessionInfo();
```

Comment:
Can we replace this `getSessionInfo` with the `getPreauthInfo` here? Better to avoid multiple api calls.

Reply:
Or update the `getSessionInfo` to call `getPreauthInfo ` internally and replace the dependency on old `/info` call